conda.recipe/meta.yaml

```diff
@@ -26,6 +26,7 @@ test:
   source_files:
     - tests
   commands:
+    - conda install -y -c conda-forge gunicorn uvicorn starlette
```

Comment:
It should be possible to declare these dependencies rather than running a command explicitly: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-requirements

Reply:
Yes, I first thought about that. Unfortunately, apparently the test requirements cannot refer to a library that is not in the same conda channel as the project; and given that uvicorn is not available in the channel, it had no way of satisfying the requirement unless I explicitly conda installed it.

Reply:
Ha ha, you can't win, sorry. No need to change then.

@jbednar @jlstevens can I change this project to use conda-forge?